baus/models.py

```diff
@@ -296,6 +302,43 @@ def jobs_relocation(jobs, employment_relocation_rates, years_per_iter,
                                 pd.Series(-1, index=index))
 
 
+@orca.step()
+def household_relocation(households, household_relocation_rates,
+                         settings, static_parcels, buildings):
+
+    # get buildings that are on those parcels
+    static_buildings = buildings.index[
+        buildings.parcel_id.isin(static_parcels)]
+
+    hh_rr = household_relocation_rates.local.copy()
+    hh_rr.loc[hh_rr.tenure == 1, 'tenure'] = 'own'
```

Comment:
@mkreilly any reason why tenure is 1/2 instead of own/rent in the rate file?  Inside urbansim it's all own/rent now.

Reply:
@fscottfoti Only because it was still 1/2 in settings.yaml before I added this. I can switch the rate file over and remove this.

Reply:
Yeah I'm for that